playbooks/roles/openvpn/templates/client-common-udp.j2

```diff
@@ -0,0 +1,30 @@
+dev tun
+proto {{ openvpn_protocol_udp }}
```

Comment:
This file is almost identical to the `client-common.j2` template. If we pull the `proto` line into the OVPN templates, then we can avoid a whole lot of duplication.